docs/tutorial-create.rst

```diff
@@ -128,21 +128,21 @@ the `Process catalog`_ of the `Resolwe Bioinformatics documentation`_.
 .. _Resolwe Bioinformatics documentation: http://resolwe-bio.readthedocs.io
 
 After uploading reads file, the next step is to align reads to a genome. We
-will use HISAT2 aligner, which is wrapped in a process with slug
-``alignment-hisat2``. Inputs and outputs of this process are described in
-`HISAT2 process catalog`_. We will define input files and the process will run
+will use STAR aligner, which is wrapped in a process with slug
+``alignment-star``. Inputs and outputs of this process are described in
+`STAR process catalog`_. We will define input files and the process will run
 its algorithm that transforms inputs into outputs.
 
-.. _HISAT2 process catalog: https://resolwe-bio.readthedocs.io/en/latest/catalog-definitions.html#process-alignment-hisat2
+.. _STAR process catalog: https://resolwe-bio.readthedocs.io/en/latest/catalog-definitions.html#process-alignment-star
 
 .. literalinclude:: files/tutorial-create.py
    :lines: 67-76
 
 Lets take a closer look to the code above. We defined the alignment process, by
-its slug ``'alignment-hisat2'``. For inputs we defined data objects ``reads``
+its slug ``'alignment-star'``. For inputs we defined data objects ``reads``
 and ``genome``. ``Reads`` object was created with 'upload-fastq-single'
 process, while ``genome`` data object was already on the server and we just
-used its slug to identify it. The ``alignment-hisat2`` processor will
+used its slug to identify it. The ``alignment-star`` processor will
 automatically take the right files from data objects, specified in inputs and
 create output files: ``bam`` alignment file, ``bai`` index and some more...
 
```

Comment:
Is it possible to have a General RNA-seq pipeline as a workflow, instead of BBDuk-STAR-HTSeq-count which is rarely used?